assets/js/job-submission.js

```diff
@@ -50,6 +50,21 @@ jQuery(document).ready(function($) {
 
 	$( '.fieldset-type-file input[type=file][multiple][data-file_limit]' ).trigger( 'update_status' );
 
+
+	$( document.body ).on( 'click', '#submit-job-form .button.save_draft', function() {
+		var $form    = $(this).closest( '#submit-job-form' );
+		var is_valid = true;
+
+		$form.find( 'div.draft-required input[required]').each( function() {
```

Comment:
Do you have a way to test this? When I inspect the HTML I don't see any fields with a class of `draft-required`.

Reply:
It is on the registration fields. You have to be signed out.

Reply:
Cool, thanks.

When I leave the required fields blank and try to save the draft, I get the validation notice for the _last_ required field in the form. This is backwards from the default browser behaviour, where we get the validation for the _first_ required field. Is this something we can fix?

Reply:
Yup f3ab44aeed6871022a0a08cb73c548444071adba